eng/tools/spec-gen-sdk-runner/src/commands.ts

```diff
@@ -86,6 +86,8 @@ export async function generateSdkForSpecPr(): Promise<number> {
   let currentRunHasBreakingChange = false;
   let overallExecutionResult = "";
   let currentExecutionResult = "";
+  let stagedArtifactsFolder : string | undefined;
```

Comment:
Can you just initialize `stagedArtifactsFolder` as "" as there is empty string check when use it in the pipeline?